services/src/main/scala/cromwell/pipeline/service/ProjectService.scala

```diff
@@ -29,9 +30,12 @@ trait ProjectService {
 object ProjectService {
 
   object Exceptions {
-    final case class ProjectNotFoundException(message: String = "Project not found") extends RuntimeException(message)
-    final case class ProjectAccessDeniedException(message: String = "Access denied. You  not owner of the project")
-        extends RuntimeException(message)
+    sealed abstract class ProjectServiceException(message: String) extends ServiceException(message)
+
+    final case class NotFound(message: String = "Project not found") extends ProjectServiceException(message)
+    final case class AccessDenied(message: String = "Access denied. You not owner of the project")
```

Comment:
Maybe it is a proper time to change that "heavy accent" exception message. How about "Wrong owner", "You are not the project owner", "Wrong user", "It's none of your business! Go away"

Reply:
Are you joking ?

Reply:
Partly :) "You not owner" really hurts.

Reply:
Ok, done.